libc/src/__support/fixed_point/fx_bits.h

```diff
@@ -163,6 +163,10 @@ template <typename T> LIBC_INLINE constexpr T round(T x, int n) {
   return bit_and((x + round_bit), rounding_mask);
 }
 
+template <typename T, typename XType> LIBC_INLINE constexpr T fx_bits(XType x) {
+  return cpp::bit_cast<T, XType>(x);
+}
+
```

Comment:
instead of defining a new function, you can just use `FXBits<T> bits(x); return bits.get_val<some_int_type>();` in the functions below

Reply:
The `FXBits<T>(XType x)` constructor checks if `XType` and `T` are the same type, so it wouldn't compile if passed `some_int_type` as x. Correct me if I'm wrong though.

I could add another constructor but that feels a little too hefty for this series of functions. 

Let me know what you think.

Reply:
I don't think adding an int constructor would be too difficult, it should just be checking that the input type is the same as `StorageType`, but there's another possibility:

This function is equivalent to just calling `bit_cast` directly, so you could replace all of the uses in the other functions with `bit_cast`.